### PR TITLE
fix the verification email request type

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/services/users/users.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export default class UserSdk {
 
   createUser(identityToken: string): Promise<CreateUserResponse> {
     return this.axiosClient.post<CreateUserResponse>(
-      `/users`,
+      `/users/users`,
```
Comment (tboskkk) [inline src/services/users/users.ts:52]: `users/users`? LOL

**Comment by tboskkk** [comment]
Looks good aside from curiosity for the naming :p

**Review by tboskkk — APPROVED** [review]

File: src/services/users/users.ts (hunk 1 above)
Comment (ghost) [reply src/services/users/users.ts:52]: Bahahah yeah... its an old service LOL

Comment (ghost) [reply src/services/users/users.ts:52]: Its going to be refactored in the future, but for now this is what we got 😂

**Pull request merged**